geth-primitives/deposit/contract.go

```diff
@@ -22,4 +22,4 @@ package deposit
 
 // TODO: Remove ldflags=-checklinkname=0 override once fix is applied.
 //
-//go:generate go run -ldflags=-checklinkname=0 github.com/ethereum/go-ethereum/cmd/abigen  --abi=../../../../contracts/out/PermissionedDepositContract.sol/PermissionedDepositContract.abi.json --bin=../../../../contracts/out/PermissionedDepositContract.sol/PermissionedDepositContract.bin --pkg=deposit --type=BeaconDepositContract --out=contract.abigen.go
```

Comment:
@fridrik01 nice, how did you catch this?

Reply:
There were some many little things that were not working, but IIRC I just did a global "../../../" search and manually made sure the relative path made sense with the new directory structure